### ISSUE TITLE
Additional game rule tracking issue

### ISSUE THREAD

**Issue by gegy1000** [NucleoidMC/plasmid#41]
This is an issue to track all game rules that we want to support in Plasmid:
- [ ] Mutating inventory
- [ ] Player movement
- [ ] Player visibility
- [ ] Player collisions
- [ ] Name tag visibility

**Comment by Restioson** [comment]
With the one on player collisions, would an internal team be created to facilitate this or would another strategy be used? How does this interact with other games using teams?

**Comment by gegy1000** [comment]
@Restioson Yeah, that will be needed. Not totally sure how we should do that, but we'll definitely need to do some design work to figure out how to set that out. Another problem is creating/disposing the teams when the game opens/closes, or if the rule were to be changed.
